plugins/generator-1.20.6/neoforge-1.20.6/mappings/stepsounds.yaml

```diff
@@ -108,4 +108,7 @@ DECORATED_POT: DECORATED_POT
 DECORATED_POT_CRACKED: DECORATED_POT_CRACKED
 TRIAL_SPAWNER: TRIAL_SPAWNER
 SPONGE: SPONGE
-WET_SPONGE: WET_SPONGE
\ No newline at end of file
+WET_SPONGE: WET_SPONGE
+VAULT: VAULT
+HEAVY_CORE: HEAVY_CORE
+COBWEB: COBWEB
```

Comment:
COBWEB is not positioned here in the code, please fix this.

Reply:
but don't I have to keep the order in the Minecraft file?

Reply:
This is the goal of mappings, _they should be ordered same as in vanilla code_

Reply:
but this is how it appears in vanilla code (it is the last sound type added)
![image](https://github.com/MCreator/MCreator/assets/105946758/3f8a1656-1cca-4430-a067-0ac7bd6fedf3)

Reply:
I see. I was looking at wrong sources (the ones auto-provided by IDEA)